SS14.Launcher/Views/MainWindowTabs/OptionsTabView.xaml.cs

```diff
@@ -24,4 +26,16 @@ public OptionsTabView()
             DispatcherTimer.RunOnce(() => { window.Classes.Remove("DoAFlip"); }, TimeSpan.FromSeconds(1));
         });
     }
+
+    public async void ClearEnginesPressed(object? _1, RoutedEventArgs _2)
+    {
+        (DataContext as OptionsTabViewModel)!.ClearEngines();
```

Comment:
Would like some advice on the `async void` here (I think it has to be `void` because of the type signature required for `Click`, as well as using the exclamation mark for the cast, as I figured the error it woudl spit out if something went wrong would probably be more useful than anything I would log instead.

Reply:
The void thing is fine. Use this as cast though.

```suggestion
    public async void ClearEnginesPressed(object? _1, RoutedEventArgs _2)
    {
        ((OptionsTabViewModel)DataContext)!.ClearEngines();
```